modules/manage/pages/manage-expiration.adoc

```diff
@@ -146,8 +149,77 @@ You can also use the Web Console to change the `maxTTL` setting of an existing b
 , If you're enabling automatic expiration, enter the number of seconds the documents should exist before expiring in the text field.
 . Click btn:[Save Changes].
 
-You cannot use the Web Console to change the `maxTTL` of a collection after ypu create it.
+You cannot change the `maxTTL` of a collection after you create it.
  
-== Set maxTTL Using the REST API
+=== Set maxTTL Using the REST API
+
+The REST API has endpoints for creating and editing collections and buckets. These endpoints let you set a collection or bucket's the `maxTTL` setting on creation.
+
+The following example gets the value of `maxTTL` of a bucket named `user_sessions`, then sets it using the `/pools/default/buckets` endpoint:
+
+[source, shell]
+----
+curl -s -X GET -u Administrator:password \
+      http://localhost:8091/pools/default/buckets/user_sessions  \
+      | jq '{maxTTL: .maxTTL}'
+{
+  "maxTTL": 0
+}
+
+curl -X POST -u Administrator:password \
+      http://localhost:8091/pools/default/buckets/user_sessions \
+      -d maxTTL=7200
+
+curl -s -X GET -u Administrator:password \
+      http://localhost:8091/pools/default/buckets/user_sessions \
+      | jq '{maxTTL: .maxTTL}'
+{
+  "maxTTL": 7200
+}
+----
+
+See xref:rest-api:rest-bucket-create.adoc[] for more about editing buckets.
+
+You can only set the `maxTTL` of a collection when creating it. The following example creates a collection named `test_site` in the `store` scope of the `user_sessions` bucket and sets its `maxTTL` to 1 hour:
```

Comment:
maxTTL for a collection can be changed. e.g.

$ xcurl localhost:9000/pools/default/buckets/default/scopes/myscope/collections/mycollection -d 'maxTTL=123' -X PATCH

Reply:
Will adds thins in the 7.6 docs.